support/reftest/data.yml

```diff
@@ -0,0 +1,19 @@
+prod-cdn-url:
+  https://cdn.redhat.com
+
+destination:
+  table: dichen-test
+  bucket: dichen-test
+
+path:
+  # listing file
+  - /content/dist/rhel/server/5/listing
+  - /content/dist/rhel/server/5/5.7/listing
+  # packages
+  - /content/aus/rhel/server/6/6.5/x86_64/os/Packages/c/cpio-2.10-12.el6_5.x86_64.rpm
+  - /content/dist/rhel/server/7/7.4/x86_64/os/Packages/c/cloud-init-0.7.9-9.el7.x86_64.rpm
+  # repodata
+  - /content/dist/rhel/server/7/7.4/x86_64/os/repodata/repomd.xml
+  - /content/dist/rhel/server/7/7.4/x86_64/os/repodata/fd23895d43f54a50bbd0509809dd5f45298bfd6b-other.sqlite.bz2
```

Comment:
To elaborate on what I said elsewhere about storing sums and content types in the testdata rather than calculating dynamically.  I think this would be better defined as something like this:

```
- path: /content/dist/rhel/server/7/7.4/x86_64/os/repodata/fd23895d43f54a50bbd0509809dd5f45298bfd6b-other.sqlite.bz2
  sha256: 647b38c2222caecd0859f97a0ef0ef78f249b8d96af437552d6b2950056f261e
  content-type: application/x-bzip
- path: /content/dist/rhel/server/7/7.4/x86_64/os/repodata/cb753af26534673064bd593500d747d7288d75b2-filelists.xml.gz
  sha256: 49159602b5bb8d9d34af384a62bd411d12449338501a4d08ef90c369b7facb98
  content-type: application/x-gzip
```

So the reference testdata actually defines the most relevant properties of the test data right there, rather than calculating it on the fly with code which could be buggy and would need to be maintained in sync with publishing tool code.